gitops/snapshot.go

```diff
@@ -775,3 +775,33 @@ func ResetSnapshotStatusConditions(adapterClient client.Client, ctx context.Cont
 
 	return nil
 }
+
+// CopySnapshotLabelsAndAnnotation coppies labels and annotations from build pipelineRun or tested snapshot
+// into regular or composite snapshot
+func CopySnapshotLabelsAndAnnotation(application *applicationapiv1alpha1.Application, snapshot *applicationapiv1alpha1.Snapshot, componentName string, source *metav1.ObjectMeta, destination *metav1.ObjectMeta, prefix string, isComposite bool) {
```

Comment:
but `destination` is clear here, it's always `snapshot` why do we need to list it explicitly as a parameter?